lambdas/service/app.py

```diff
@@ -27,11 +27,6 @@
 # how-to-write-docstring-for-url-parameters
 
 
-@app.route('/', cors=True)
```

Comment:
One of the tests relies on this:

hca/azul/test/service/request_validation_test.py